puzzles/is-palindrome/isPalindrome.js

```diff
@@ -0,0 +1,26 @@
+// -----
+// Run time env (tested on):
+// Node v11.12.0
+// -----
+// Algorithm complexity
```

Comment:
Every solution that is submitted needs to contain its algorithm complexity value.